### DISCUSSION TITLE
Task execution order on deletion/removal

### DISCUSSION THREAD

**Discussion by IskanderNovena** [org-formation/org-formation-cli#427] [Q&A]
How can I make sure that on removal of a task, the sub-tasks are executed in reversed order of when adding the task when using `update-stack` tasks?

So far I've tried it with setting the `DependsOn` pointing to the previous task(s), but that doesn't result on actually removing the last task before the previous one

**Comment by project0** [comment]
I have the same problem, any idea how to fix this?
